alertmanager/config/config.go

```diff
@@ -84,6 +85,51 @@ func (c *Config) Validate() error {
 	return nil
 }
 
+func (c *Config) SearchRoutesForReceiver(receiver string) bool {
+	if c.Route.Receiver == receiver {
+		return true
+	}
+	for _, route := range c.Route.Routes {
```

Comment:
`func.Contains`?

Reply:
this is a recursive struct. funk.Contains wouldn't work here

Reply:
I guess you could use funk.Filter() and then check the length of the filtered array but at that point you're just making it more complicated than vanilla Go